pages/index.tsx

```diff
@@ -59,7 +59,7 @@ export default function Index() {
     options.push("yUSD Yield");
   }
 
-  if (weth && collAddress == weth.address) {
+  if (weth && collAddress?.toLowerCase() == weth.address.toLowerCase()) {
```

Comment:
I haven actually used this new feature yet. What is this doing exactly, only calling toLower if collAddress it not null?

Reply:
yep, otherwise the LHS of this equation will be `undefined`